src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Clears all persons and tags from the address book.
```

Comment:
But it wouldn't be that good if the comments and command are not aligned. Clears =/= `sort` XD